### PR TITLE
Fixes#25 Added Font family OpenSans

### LINKED ISSUE #25: Add font family.
Add `Open Sans` font family for global styling.
Steps for creation:

- Create a new file `fonts.scss` for this addition
- Import it in base.scss.

Here is the reference link https://fonts.google.com/specimen/Open+Sans
Please let us know in case of any query.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #25 

#### Changes proposed in this pull request:

- Added Font family OpenSans

### CODE REVIEW

**Comment by ParthS007** [comment]
@aka-jain Please review . Thanks

**Comment by ParthS007** [comment]
I have gone through Travis Logs and I don't think that Travis has failed due to this PR. @aka-jain  Please help and suggest changes if any.

**Comment by ParthS007** [comment]
@aka-jain  waiting for your review :slightly_smiling_face:

File: src/styles/base.scss [hunk 1]
```diff
@@ -35,3 +35,105 @@
 .fs-36 {
   font-size: $fs-36;
 }
+
+/*Font Faces*/
```
Comment (aka-jain) [inline src/styles/base.scss]: Make a new file for this. `font.scss` @ParthS007

**Comment by ParthS007** [comment]
@aka-jain  I have made the changes, Please review. Thanks :slightly_smiling_face:

**Comment by ParthS007** [comment]
@aka-jain  waiting for your review :slightly_smiling_face:

**Comment by ParthS007** [comment]
@aka-jain waiting for your review

**Comment by aka-jain** [comment]
Will do it @ParthS007.

**Comment by ParthS007** [comment]
@spyshiv can you please restart the Travis build  of this PR.

**Comment by spyshiv** [comment]
@ParthS007 I tried. trvis.yml is not syncing with master branch that's why still not working. May be need further commit in this PR to trigger updated travis.yml

**Comment by ParthS007** [comment]
@spyshiv  Have you tried by clicking on this 
image
present at the rightmost position of Travis PR dashboard. 
I hope this will work.
I can close and reopen it for restarting Travis.

**Comment by ParthS007** [comment]
@aka-jain @spyshiv Please review. Passing all the tests :slightly_smiling_face:

**Review by aka-jain — APPROVED** [review]

**Pull request merged**